### PR TITLE
Feat #1763/remove unused libs  master

### LINKED ISSUE #1763: [FEATURE] Cleanup project dependencies
**Describe how this feature will help us improve**
Describe why do you think this feature is needed. 

`package.json` contains dependencies that are not used anymore (like `vuepress` or I think `axios`). Some depencies can possibly be moved to devDependencies. Also some dependencies are used only in one case (i.e. `vue-fragment`) so they can be replaced.

**Is your proposal related to a problem?**
Describe the issue or type its number

**Describe the solution you'd considered**
A concise description of your preferred solution. Things to address include:

  - Details of the technical implementation
  - Tradeoffs made in design decisions
  - Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.

**Additional context**
Anything you would like to add but was not sure which section it fits.

### PR DESCRIPTION
**Pull request by Baroshem**
# Related issue

Closes #1763 

# Scope of work

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Review by justyna-13 — CHANGES_REQUESTED** [review]
I think something went wrong with tabs :) 
Screenshot from 2021-05-06 14-00-47
Also would be good not to lose a nice look of scrollbar :) 
Screenshot from 2021-05-06 14-02-31

**Review by AdamPawlinski — COMMENTED** [review]
Just a few questions.
Regarding scrollable, can we get rid of the simplebar import from scss and `simplebar-scrollbar:before` properties as they are not longer needed? And I think, like Justyna, that we can make another issue for styling scrollbar to leave it like before. 

We could also remove folder with vuepress stuff, wdyt? 
image


I wonder about adding vue as the dependency, if it's necessary, can we add it to devDependencies?

File: packages/shared/styles/components/molecules/SfScrollable.scss [hunk 1]
```diff
@@ -19,4 +19,26 @@
   &__view-all {
     margin: var(--scrollable-button-margin, var(--spacer-sm) 0 0 0);
   }
+
+  &::-webkit-scrollbar {
+    background-color: #fff;
```
Comment (justyna-13) [inline packages/shared/styles/components/molecules/SfScrollable.scss:24]: `--c-white`

File: packages/shared/styles/components/molecules/SfScrollable.scss [hunk 2]
```diff
@@ -19,4 +19,26 @@
...
+    width: 16px;
```
Comment (justyna-13) [inline packages/shared/styles/components/molecules/SfScrollable.scss]: `--spacer-sm`

**Review by justyna-13 — APPROVED** [review]
:+1:

**Pull request merged**